datafusion/expr/src/logical_plan/builder.rs

```diff
@@ -119,6 +121,7 @@ impl LogicalPlanBuilder {
     /// The column names are not specified by the SQL standard and different database systems do it differently,
     /// so it's usually better to override the default names with a table alias list.
     pub fn values(mut values: Vec<Vec<Expr>>) -> Result<Self> {
+        // todo: hanlde for Placeholder expr
```

Comment:
yeah, in general the Values statement should probably be cleaned up / generalized. I can try to help with this next week if that is useful